### ISSUE TITLE
Pressure tanks not working.

### ISSUE THREAD

**Issue by FlattestGuitar** [ParadiseSS13/Paradise#4102]
**Problem Description**:
The big plasma/oxygen pressure tanks cannot have their contents distributed.

**What did you expect to happen**: 
Well, they are there for a reason, right?

**What happened instead**:
Nothing happens. The gas doesn't come out.

**Why is this bad/What are the consequences:**
They should be useful.

**Steps to reproduce the problem**:
I tried to reproduce the issue and asked around if there was anything I had missed. Everything worked the same every time and it seems like me missing something is not the case.

![](https://i.gyazo.com/a15789a2224e3cf05fe9c0402bf98832.png)

The configuration on the picture pumps no gas into the yellow canister. The red pressure tank still has plasma in it, it's been untouched the whole shift.

**Comment by FalseIncarnate** [comment]
I've been noticing a number of atmos-related stuff not working properly lately, including the pressure cannisters (not a HUGE deal in my opinion, since I normally pipe in some connections to main atmos anyways when working with the incinerator / turbine). Some others include new vents and gas mixers not doing their jobs despite having power and being on and such. This is likely related, since it's stuff that probably just isn't transferring the air datums properly/at all.

**Comment by Bigtimetripod** [comment]
LINDA has left its mark and is seeking revenge!

**Comment by CrAzYPiLoT-SS13** [comment]
Might as well leave this here. On a number of rounds I played as an atmos
tech, when I fiddled with the system, I turned off the Waste to Filter pump
for about a minute to add some pipes. Afterwards, I turned it on, and for
some reason, when I checked the atmos console, the distro was losing
pressure, which was confirmed by the Filter to AirMix meter which was
telling me Filter was giving off smaller amounts of gas. I haven't actually
touched the AirMix or the distributer canisters apart from setting the
pumps to 4500kPa, so I guess it shouldn't have affected the pressure so
much, if only the WF pump was turned off. I did actually revert to the
starting setup after some time, and the pressure was still dropping. I
tried the same thing on two separate rounds, as it was actually safe enough
to do that, because the pressure wasn't dropping fast enough to actually
put the station in danger before the end of the round.

TL;DR Turned off Waste to Filter, changed some pipes, turned back on,
pressure was dropping, reverted to starting setup, pressure still dropping.

**Comment by Calecute** [comment]
Could reproduce, but it looks like tanks are working fine, what does not work is the pump. The pipes before the pump are filled, and if you remove the pump and put a pipe the plasma tank fills the cannister. Probably related to other pumps not working bugs, or something in #8245

![image](https://user-images.githubusercontent.com/34964590/55268507-e6934b80-5268-11e9-9b9c-dfef32a4b085.png)

**Comment by Spaghetti-bit** [comment]
![image](https://github.com/ParadiseSS13/Paradise/assets/60483458/e60b0613-e260-431f-aa90-6192ea91ad5a)
![image](https://github.com/ParadiseSS13/Paradise/assets/60483458/e12fb512-c50c-4ff5-8696-95c97690e802)

Fixed, likely by MILLA. Closing.
